### PR TITLE
[36] Minor UI Requirements

### PR DESCRIPTION
**Pull request by Drish-xD**
TODOs: 
- Added scrollbar in dropdowns.
- added autofill platformId for drive folders.
- Rename "next" button as "Save & Next".
- Add helptext for platform field: 'Session where platform is "other" are listed under "live class" tab.'
- Replace Hint Text in time step: "Select the time period for this session to be active for each day" .

### CODE REVIEW

File: src/app/session/[type]/Steps/Basic/index.tsx [hunk 1]
```diff
@@ -43,6 +44,7 @@ const BasicForm: FC = () => {
         label: 'Platform',
         disabled: type === SessionType.EDIT,
         onValueChange: (value, form) => handleBatchFields(value, form, apiOptions, fieldsSchema),
+        helperText: `Sessions where platform is "${Platform.Others}" are listed under "Live Classes" tab.`,
```
Comment (suryabulusu) [inline src/app/session/[type]/Steps/Basic/index.tsx:47]: i think the right statement is: sessions where platform is anything other than "quiz" is listed in liveclasses tab. check for grammar here

File: src/types/form.types.ts [hunk 2]
```diff
@@ -57,7 +56,7 @@ export const basicSchema = z
         (value) => TestPlatformOptions.some((option) => option.value === value),
         'Invalid option selected'
       ),
-    name: z.string({ required_error: 'This field is required' }),
+    name: z.string({ required_error: 'This field is required' }).min(1, 'This field is required'),
```
Comment (suryabulusu) [inline src/types/form.types.ts:59]: whats this doing

**Review by suryabulusu — COMMENTED** [review]
left comments

File: src/types/form.types.ts (hunk 2 above)
Comment (Drish-xD) [reply src/types/form.types.ts:59]: I was a bug. 
we want name as mandatory field but empty string was validated previously. so I added a check that it must have atleast 1 char in input.

File: src/app/(home)/ModalData.ts [hunk 3]
```diff
@@ -106,7 +106,6 @@ export const displayData = (data: Session, formOptions: Option[]) => {
   const liveDetails: DataSection = {
     title: 'Session Details',
     data: [
-      { label: 'Session Name', value: data.name },
```
[the remark this answers is not in the document]
Comment (Drish-xD) [reply src/app/(home)/ModalData.ts:109]: It's already in the 'Basic Details' section. It's duplicate in this section also.

File: src/app/session/[type]/Steps/Basic/index.tsx (hunk 1 above)
Comment (Drish-xD) [reply src/app/session/[type]/Steps/Basic/index.tsx]: Rephased it to this one.

**Review by suryabulusu — APPROVED** [review]

**Pull request merged**